ext/bcmath/libbcmath/src/str2num.c

```diff
@@ -38,9 +38,8 @@
 bool bc_str2num(bc_num *num, char *str, size_t scale)
```

Comment:
Does it make sense to pass the str_len here? As this should save the need to compute `strlen()` of it in the function body (also you can easily determine if we have reached the end of the str or not.

Reply:
I've already tried that and it was slower. Since the length can be obtained from zend_string, there should be no computational cost, so it's strange that it's slower....

Reply:
I would also expect that ZSTR_LEN is faster. Are you using a release build without ASAN & UBSAN instrumentation?

Reply:
Ahhh, it was a debug build.... I'll try again

Reply:
This worked quite well for the release build, but I ended up not needing the length of str, so I'll leave it unchanged.

However, I would use ZSTR_LEN if future refactoring requires it.